lib/rubocop/cop/performance/redundant_match.rb

```diff
@@ -41,21 +41,23 @@ def on_send(node)
                         !(node.parent && node.parent.block_type?)
 
           add_offense(node) do |corrector|
-            autocorrect(corrector, node)
+            autocorrect(corrector, node) if autocorrectable?(node)
           end
         end
 
         private
 
         def autocorrect(corrector, node)
-          # Regexp#match can take a second argument, but this cop doesn't
-          # register an offense in that case
```

Comment:
Can you leave the comment to `node.first_argument.regexp_type?`?

Reply:
Roger.
Actually, I don't fully understand why this comment is necessary here. Why does this comment mention the second argument? `String#match` may also take a second argument, so does it need a similar comment?

Reply:
It just keeps the status quo for future maintenance. No text updates would be needed.